hnn_core/tests/test_viz.py

```diff
@@ -47,9 +55,21 @@ def test_network_visualization():
     with pytest.raises(TypeError, match='src_gid must be an instance of'):
         plot_cell_connectivity(net, conn_idx, src_gid='blah')
 
-    with pytest.raises(ValueError, match='src_gid not in the'):
+    with pytest.raises(ValueError, match='src_gid not a valid cell ID'):
         plot_cell_connectivity(net, conn_idx, src_gid=-1)
 
+    # smoke test interactive clicking
+    del net.connectivity[-1]
```

Comment:
why is the deletion necessary?

Reply:
not strictly necessary but I thought this would be good practice, otherwise the same "kind of connection" would occur in two different `net.connectivity` elements and the net effect would be a higher weight than we intended?